tools/c7n_kube/c7n_kube/provider.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from c7n.registry import PluginRegistry
+from c7n.provider import Provider, clouds
+
+from .client import Session
+from functools import partial
+
+import logging
+
+log = logging.getLogger('custodian.kube')
+
+@clouds.register('kube')
```

Comment:
thoughts on `kube` vs `k8s` for the provider name, if we're going to use an abbreviation, i figure let's use the one common in that community.